### PR TITLE
chore: Dockerize!

### PR DESCRIPTION
**Pull request by yock**
- [x] Run Sparkbot locally inside Docker
- [ ] Have Sparkbot login to Slack from inside Docker

### CODE REVIEW

**Comment by cromwellryan** [comment]
I wonder if that login to slack part shouldn't be something we put in the readme.  You can provide a docker run command that is different than the embedde CMD.

Thoughts?

**Comment by yock** [comment]
Should we default to connecting to Slack to simplify deployment? Might be helpful to document how to set that up too. I presume it's environment variables, but I couldn't quickly figure it out.

**Comment by cromwellryan** [comment]
Oh man - i totally suck for losing track of this.  I don't think we should connect to slack by default, cause it results in this https://sparkbox.slack.com/archives/sparkbot-dev/p1472091972000005

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM debian:jessie
+
+RUN apt-get update
```
Comment (cromwellryan) [inline Dockerfile]: Consider chaining this runs like this:

```
RUN apt-get update \
    apt-get install --assume-yes nodes npm \
    ln -s /usr/bin/nodejs /usr/bin/node
```

through to the `mkdir` step.  

You could also consider using the [official node](https://hub.docker.com/_/node/) image

**Pull request closed without merging**